java/test/jmri/jmrit/beantable/beanedit/BeanEditItemTest.java

```diff
@@ -0,0 +1,39 @@
+package jmri.jmrit.beantable.beanedit;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author Paul Bender Copyright (C) 2017	
+ */
+public class BeanEditItemTest {
+
+    @Test
+    public void testCTor() {
+        BeanItemPanel p = new BeanItemPanel();
+        BeanEditItem t = new BeanEditItem(p,"IS1","");
+        Assert.assertNotNull("exists",t);
+    }
+
+    // The minimal setup for log4J
+    @Before
+    public void setUp() {
+        apps.tests.Log4JFixture.setUp();
```

Comment:
Please replace these two calls in setUp with ```jmri.util.JUnitUtil.setUp();``` and the two calls in tearDown with ```jmri.util.JUnitUtil.tearDown();```.